### PR TITLE
Removes JSONStream dependency

### PR DESCRIPTION
**Pull request by Tpt**
Works directly using jsonparse

### CODE REVIEW

File: lib/SparqlJsonParser.ts [hunk 1]
```diff
@@ -38,14 +38,26 @@ export class SparqlJsonParser {
   public parseJsonResultsStream(sparqlResponseStream: NodeJS.ReadableStream): NodeJS.ReadableStream {
     const errorListener = (error: Error) => resultStream.emit('error', error);
     sparqlResponseStream.on('error', errorListener);
-    const variables: RDF.Variable[] = [];
-    sparqlResponseStream
-      .pipe(require('JSONStream').parse('head.vars.*').on('error', errorListener))
-      .on('data', (variable: string) => variables.push(this.dataFactory.variable(variable)))
-      .on('end',  () => resultStream.emit('variables', variables));
+
+    const JsonParser = require('jsonparse');
```
Comment (rubensworks) [inline lib/SparqlJsonParser.ts:42]: Instead of using `require` here, can we move this to an `import` at the top of the file?

**Review by rubensworks — CHANGES_REQUESTED** [review]
Looks great! Just a single comment.

File: lib/SparqlJsonParser.ts (hunk 1 above)
Comment (Tpt) [reply lib/SparqlJsonParser.ts]: Sadly there is no TypeScript definitions for `jsonparse` hence the `require`. I have moved it at the top of the file.

Comment (rubensworks) [reply lib/SparqlJsonParser.ts]: Thanks :)

**Pull request merged**